### PR TITLE
proto bumps for MsgCancelUnbondingDelegation

### PR DESCRIPTION
**Pull request by turadg**
## Description

Makes the proto updating mechanism more robust, getting the latest definitions used in the SDK branch. PTAL @JimLarson 

Also adopts the new `MsgCancelUnbondingDelegation` that is available. PTAL @0xpatrickdev 

### Security Considerations

none

### Scaling Considerations

none

### Documentation Considerations

More automated now

### Testing Considerations

Manual

### Upgrade Considerations

Improves sync of cosmic-proto package with cosmic-swingset

### CODE REVIEW

File: packages/cosmic-proto/update-protos.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# update proto files in this package
+cp -rf "$COSMOS_SDK"/proto/cosmos proto
+cp -rf "$AG_SDK"/golang/cosmos/third_party/proto .
+cp -rf "$AG_SDK"/golang/cosmos/proto/agoric proto
+
+# clean up what we don't need
+chmod -R u+w proto
+rm -rf proto/cosmos/app
+rm -rf proto/cosmos/orm
```
Comment (JimLarson) [inline packages/cosmic-proto/update-protos.sh:22]: There's a lot of stuff in there that you probably don't need, and you might stub your toe a few times when needs change, but it's your call.

File: packages/cosmic-proto/MAINTAINERS.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ The generated code is determined by the contents of `protos` and the config of `
 
 ## Maintaining protos
 
-The `protos` are held in this source tree and updated from `golang/cosmos/proto` during `yarn codegen` by `yarn protos-update`. The `cosmos` protos were sourced from [@protobufjs/cosmos](https://www.npmjs.com/package/@protobufs/cosmos) per [Telescope's instructions](https://github.com/cosmology-tech/telescope?tab=readme-ov-file#add-protobufs). However we don't use that as a dependency of the package because we need a more manual approach to merge with the Golang-managed protos in the repo.
+The `protos` are held in this source tree and updated from `golang/cosmos/proto` during `yarn codegen` by the `update-protos.sh` script.
```
Comment (JimLarson) [inline packages/cosmic-proto/MAINTAINERS.md]: This is ambiguous whether `yarn codegen` invokes `update-protos.sh` or vice-versa. Make it clear that you call `yarn codegen` and consider letting folks just read the implementation if they want to see how it works.

I've updated the instructions on the [Upgrading the Interchain Stack](https://github.com/Agoric/agoric-sdk/wiki/Upgrading-the-Interchain-Stack) wiki page.

**Review by JimLarson — APPROVED** [review]
Great! This is probably sufficient for #9018 too.

File: packages/cosmic-proto/update-protos.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# go ensure fresh build
+cd "$AG_SDK"/golang/cosmos
+make all
+COSMOS_SDK=$(go list -m -f '{{ .Dir }}' github.com/cosmos/cosmos-sdk)
+cd -
+
+# update proto files in this package
+cp -rf "$COSMOS_SDK"/proto/cosmos proto
+cp -rf "$AG_SDK"/golang/cosmos/third_party/proto .
+cp -rf "$AG_SDK"/golang/cosmos/proto/agoric proto
```
Comment (0xpatrickdev) [inline packages/cosmic-proto/update-protos.sh:17]: LGTM. Seems this should preserve protos added manually (like `/icq/v1`) that are not part of `/golang/cosmos/third_party/proto`

**Review by 0xpatrickdev — APPROVED** [review]

**Pull request merged**